app/views/idv/doc_auth/upload.html.erb

```diff
@@ -19,28 +19,34 @@
 <% end %>
 
 <p>
-  <strong><%= t('doc_auth.info.upload_no_image_storage') %></strong>
+  <%= t('doc_auth.info.upload_no_image_storage') %>
 </p>
 
-<% if Figaro.env.cac_proofing_enabled == 'true' && desktop_device? %>
-  <div class='mt2 mb2'>
+<p>
+  <%= t('doc_auth.info.id_worn_html') %>
+  <a href="/"><%= t('doc_auth.info.accepted_ids') %></a>
```

Comment:
1. Can we use the `link_to` helper?
2. Should we drop a comment or something to call it out since we don't have this URL yet? Rubocop gets mad at TODO so maybe something else?

Reply:
> 1. Can we use the `link_to` helper?
> 2. Should we drop a comment or something to call it out since we don't have this URL yet? Rubocop gets mad at TODO so maybe something else?

Whoops, I totally overlooked this. It was (is) a work-in-progress, pending [a relevant Slack question](https://gsa-tts.slack.com/archives/C0NGESUN5/p1600716779044900) about where exactly this link should point to.

Reply:
Some more quick thoughts on this:
1. Can we add a method in our [MarketingSite](https://github.com/18F/identity-idp/blob/master/app/services/marketing_site.rb) class for it?
2. Should we expose this URL as yet another data attribute to the React app to minimize duplication?

Reply:
> Some more quick thoughts on this:
> 
> 1. Can we add a method in our [MarketingSite](https://github.com/18F/identity-idp/blob/master/app/services/marketing_site.rb) class for it?
> 2. Should we expose this URL as yet another data attribute to the React app to minimize duplication?

Yeah, I'd been waiting to see how exactly we wanted to approach this before coming to any decision on implementation. But if we're going to direct users to the marketing site, I agree it makes sense to add a method there. I'd also agree that it makes sense to expose it to the React app as a `data-` attribute. I'm not quite as sure how best to actually make this data available to the component, aside from introducing yet another context provider, possibly mapping directly to this `MarketingSite` class. It doesn't really fit well into any of the existing context values.

It's also got me thinking a bit about future re-use. The idea of a React context which captures some or all of the values from the `MarketingSite` class seems clean enough and ripe for reuse, though unclear right now to what extent we want to go down this path (which methods? which package? or new package(s)? etc.)

Reply:
> I'm not quite as sure how best to actually make this data available to the component, aside from introducing yet another context provider, possibly mapping directly to this `MarketingSite` class. It doesn't really fit well into any of the existing context values.

Is it worth just collapsing all our existing contexts into "DocAuthContext" or something? We only have one React app/page today?

Is there any performance penalty to having too many small contexts?

Reply:
> Is it worth just collapsing all our existing contexts into "DocAuthContext" or something? We only have one React app/page today?

We've finally found our way to the eternal debate of multiple contexts vs. global store (Redux) 😄   We could certainly have opted to just shove everything into a single global state object. At our current position, I'd personally still lean toward pushing forward and continuing to use distinct contexts. There's a bit more overhead with managing each one, but it forces us to be somewhat more intentional, and potentially opens up some options for reuse in the future.

Short-term, I imagine this would look like: `MarketingSiteContext` in the same pattern as other contexts, likely only with a single property for the "Acceptable IDs" link.
 
> Is there any performance penalty to having too many small contexts?

Not that I'm aware of, or at least not one I'd be particularly concerned about at our scale. Usually it's the opposite, where contexts have an advantage over global store approaches where global stores cause lots of unnecessary re-renders of components which ultimately aren't concerned with the change and won't have a changed rendered result ([example](https://github.com/WordPress/gutenberg/pull/12877)).